### PR TITLE
lint: Increase linter coverage of test code to cover decorators, libvmops and usb

### PR DESCRIPTION
**Pull request by brianmcarey**
### What this PR does
Before this PR:

After this PR:

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Special notes for your reviewer

/cc @dhiller @xpivarc 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/libvmops/lifecycle.go [hunk 1]
```diff
@@ -30,13 +30,15 @@ import (
 	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
-	. "kubevirt.io/kubevirt/tests/framework/matcher"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
 
 	v1 "kubevirt.io/api/core/v1"
 )
 
+const stopVMTimeout = 300
```
Comment (fossedihelm) [inline tests/libvmops/lifecycle.go:38]: This const can be defined inside `StopVirtualMachine`

File: tests/usb/usb.go [hunk 2]
```diff
@@ -32,6 +32,7 @@ import (
 const (
 	failedDeleteVMI = "Failed to delete VMI"
 	cmdNumberUSBs   = "dmesg | grep -c idVendor=46f4"
+	deleteTimeout   = 180
```
Comment (fossedihelm) [inline tests/usb/usb.go:35]: can be defined in the afterEach

**Review by fossedihelm — COMMENTED** [review]
Thank you!
some small comments

**Comment by brianmcarey** [comment]
> Thank you! some small comments

Thanks @fossedihelm for the review - I have addressed your comments.

**Review by fossedihelm — APPROVED** [review]
Thank you!

**Review by EdDev — APPROVED** [review]
This is great, thanks!

**Review by enp0s3 — COMMENTED** [review]
/approve
@brianmcarey Thank you!

**Comment by brianmcarey** [comment]
/override pull-kubevirt-e2e-k8s-1.31-sig-performance

Triggered by status-reconciler when lane was bumped.

**Pull request merged**